Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -53,8 +53,6 @@
       - !type:PlaySoundBehavior
         sound:
           collection: MetalGlassBreak
-  - type: Repairable
```

Comment:
i think this should still use Repairable but implement a minDamage field that only starts the doafter if theres at least that much GetTotal() damage